sre_maturity.md

```diff
@@ -121,21 +121,31 @@ Chaos testing in production during the available time window while still meeting
 
 ### Security - Phase 1: The ‘Crawl’ Phase
 
-There is no need to get granular with authz policies to start with, but do start with production access restricted to a tightly scoped set of individuals.
+Security are the controls (administrative, technical, physical) that are put in place to manage risk of a company to an acceptable level.
+At this phase you are not granular with authz policies to start with, but do have production access restricted to a tightly scoped set of individuals.
 Granularity can come later with things like audited access, permissions escalation & tiered access as you mature the debugging experience of your service.
 You should trust your team and assume positive intent.
 
-In this phase you do ad-hoc security assessments, scans and pentesting.
+You have a mecahnismm for managing credentials & secrets, and deliverying them as configuration into your services.
+Additionally, you have a credential rotation policy that's based on time and individuals leaving your team.
+
+You are likely doing ad-hoc security activities such as assessments, vulnerability scans and pentesting.
+These ad-hoc tests inform larger release milestones rather than continious releasing.
 
 ### Security - Phase 2: The 'Walk' Phase
 
-*This section needs elaboration. Please help by adding content. Ref:[SIGSRE-84](https://issues.redhat.com/browse/SIGSRE-84)*
+In this phase you start to look at compliance.
+Compliance relates to all the activities implemented to make sure that security controls meet requirements from compliance certifications/attestations (i.e. ISO 27001:2013, HIPAA, SOC 2 Type 2, PCI DSS, etc.)
+Those compliance certifications/attestations are the tangible items you can give to your customers to provide assurances of your security posture.
```

Comment:
It's important to include things like ESS, which is an internal compliance process.

Reply:
It would be helpful to link to a public-facing doc on what ESS is. I'm not sure if it is well known outside of Red Hat

Reply:
I'm not aware of a public facing doc. It seems to be internally focused and driven only

Reply:
In that case, unless we plan to publish it, what do you think about either skipping the reference or adding a few sentences clarifying that this is in an internal framework?

Reply:
Great.
ESS isn't mentioned anyways in the PR, so I'd prefer to keep it out.

This line hints at it, but is vague enough to ask the reader a question i.e. does their company have a process they should consider here.
`Companies may also have your own internal compliance processes or security standards`